### PR TITLE
Git-101 Practice

### PR DESCRIPTION
**Pull request by anna-blumenkrantz**

### CODE REVIEW

File: .history/README_20220202103731.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Students Repo
```
Comment (nchursin) [inline .history/README_20220202103731.md]: Please, remove this file

File: .history/README_20220202111053.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Students Repo
```
Comment (nchursin) [inline .history/README_20220202111053.md]: Please, remove this file

File: .history/students/students1_20220202103731.list [hunk 3]
```diff
@@ -0,0 +1,4 @@
+Student name,Student Email
```
Comment (nchursin) [inline .history/students/students1_20220202103731.list]: Please, remove this file

File: .history/students/students1_20220202104238.list [hunk 4]
```diff
@@ -0,0 +1,4 @@
+Anna Blumenkrantz,sewingrus@gmail.com
```
Comment (nchursin) [inline .history/students/students1_20220202104238.list]: Please, remove this file

File: .history/students/students1_20220202111107.list [hunk 5]
```diff
@@ -0,0 +1 @@
+Anna Blumenkrantz,sewingrus@gmail.com
```
Comment (nchursin) [inline .history/students/students1_20220202111107.list]: Please, remove this file

File: README.md [hunk 6]
```diff
@@ -9,6 +9,10 @@ To pass the Git-101 challenge you need to get a Pull Request merged into this re
 Acceptance Criteria:
 
 1. Create a new branch in [this repo](https://github.com/wedoforce/students) called `student/<first name>-<last name>`. E.g. `student/nikita-chursin`
-1. Pull request (PR) is raised against the `main` branch
+Created branch student/anna-blumenkrantz
```
Comment (nchursin) [inline README.md]: Please revert these changes to the main branch files. You can easily do this via the following command `git checkout main README.md` and then commit changes

File: students/students1.list [hunk 7]
```diff
@@ -1,4 +1 @@
-Student name,Student Email
-Serhii Zhuhan
-Oleg Mastriukov
-
+Anna Blumenkrantz,sewingrus@gmail.com
```
Comment (nchursin) [inline students/students1.list]: This file has conflicts. Please resolve them. For theory about conflicts please refer to [merge-conflicts.md](https://github.com/wedoforce/sf-dev-101-practice/blob/lessons/git-101/merge-confilcts.md)

**Review by nchursin — CHANGES_REQUESTED** [review]

File: students/students1.list (hunk 7 above)
Comment (anna-blumenkrantz) [reply students/students1.list]: at the time of merge the branch was already up to date.

**Comment by anna-blumenkrantz** [comment]
git merge main says Already up to date.

**Comment by nchursin** [comment]
I believe you did something like this:
1. You're on your branch
2. `git fetch` or `git pull`
3. `git merge main`

This is something we probably should cover... I just thought it's too advanced for 101.

The problem is `main` is in your local repo and it remains unchanged. `git pull` only pulls changes to your current branch. `get fetch` in turn only fetches changes to *your local copy of a remote repo* but *not your local working repo*. 

To actually merge the *remote copy of the branch* you need to execute `git merge origin/main`. If you take a closer look at the [cheat sheet](https://github.com/wedoforce/sf-dev-101-practice/blob/lessons/git-101/merge-confilcts.md#cheat-sheet) in `merge-conflicts.md` you could see that it shows the command to merge branch from a remote.

**Comment by anna-blumenkrantz** [comment]
Conflict merged

**Review by nchursin — APPROVED** [review]
